### ISSUE TITLE
Feature request: Parameters compare

### ISSUE THREAD

**Issue by kaklik** [PX4/flight_review#191]
For logfile analysis is a usual task to verify parametric setup between multiple flights.  The current approach by manual comparing of parameters between multiple versions of firmware (Where parameters set is not the same) is somewhat time-consuming. 

The good solution to that problem would be expanding the flight_review by parameter comparison tool. 

It should work by one of those ways: 

  1. From currently opened log file select a compare tool from top. 
  2.  Log browser should be opened  (currently implemented)
  3. select the other already uploaded log file to compare. 

The another option 

1. In the current existing log-browser implement the compare checkbox. 
2. User select checkbox of multiple logs 
3. clicks on the "compare logs" button. 

In both cases, the table with different parameters should be displayed. 
A nice option would be filtering of obviously different parameters, like battery calibration, RC mapping, status parameters, and sensor ids, or calibration.

**Comment by bkueng** [comment]
I agree it is a nice feature. Requires some work both on the UI and backend though.

As usual: PR's are welcome.
